package/ota_daemon/src/ota_daemon.c

```diff
@@ -362,7 +359,7 @@ static bool ota_client_loop(void)
     ota_enquire(nw_ctx);
 
     ota_resp_t parsed_resp = {0};
-    if (!ota_parse_response(&parsed_resp)) {
+    if (libnetwork_shutdown_signaled(nw_ctx) || !ota_parse_response(&parsed_resp)) {
```

Comment:
I added these in case the shutdown is requested during download. For example in the case of `ota_download()` it would proceed to sha256 calculation and give sha mismatch errors that could confuse user.

But it turns out that with the return value additions in the libnetwork byte limit PR these checks become redundant so I will remove these.